fab/inventory/production

```diff
@@ -60,5 +62,5 @@ datavol_device=/dev/xvdb
 [stanchion]
 hqriak0.internal-va.commcarehq.org
 
-[rackspace_cloud_backup]
-hqdb0.internal.commcarehq.org
+# [rackspace_cloud_backup]
+# hqdb0.internal-va.commcarehq.org
```

Comment:
You're right, I was thinking of the rackspace feature itself, not the corresponding ansible command.  I just merged your PRs getting rid of the feature and this inventory block, which isn't needed.